src/components/input/math-input.js

```diff
@@ -208,6 +210,7 @@ class MathInput extends React.Component {
         window.removeEventListener('resize', this._clearKeypadBoundsCache());
         window.removeEventListener(
                 'orientationchange', this._clearKeypadBoundsCache());
+        window.document.addEventListener("focusout", this._keepInputFocus);
```

Comment:
Should this line be `window.removeEventListener("focusout", this._keepInputFocus);`?

Reply:
Yep. :( Fixed!